package-lock.json

```diff
@@ -1028,85 +1028,6 @@
         "through2": "^2.0.3"
       }
     },
-    "@istanbuljs/load-nyc-config": {
```

Comment:
Es ist ganz schön viel aus der package-lock.json raus geflogen. Löst die common die Äbhängigkeiten jetzt anders auf?

Reply:
in der commons habe ich etwas aufgeräumt... gut möglich dass das damit zusammenhängt

Reply:
ok